### DISCUSSION TITLE
AMP Viewport Meta Overwrite query

### DISCUSSION THREAD

**Discussion by tapan-thapa** [vercel/next.js#26879] [Help]
Hi Team,

By default, nextjs is adding viewport meta as below for AMP pages.

`<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">`

I want to overwrite like below.

`<meta name="viewport" content="width=device-width">`

Please suggest how to do it.

BR
Tapan Thapa

**Comment by tapan-thapa** [comment]
Any help here will be greatly appreciated.

**Comment by tapan-thapa** [comment]
Any help please

**Comment by deepanswer** [comment]
I am struggling with the same problem. 

Using amp with nextjs generates a meta viewport that can't be changed, `<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">`

Google is using [The AMP page Experience](https://amp.dev/about/page-experience/) to reference in the **Google Search Console** for the Core Vitals as a way to evaluate search results.

When check an "amp only" nextjs page using The AMP page experience shows **Disable Tap Delay warning** even if the page has a viewport meta with property `width=device-width`.

No Issues : `<meta name="viewport" content="width=device-width">`
Displays tap delay Warnning : `<meta name="viewport" content="width=device-width,initial-scale=1">`

While being valid AMP, it turns out that a meta viewport like this causes a tap delay (impacting FID):

`<meta name="viewport" content="width=device-width, initial-scale=1" />`

The only solution is to change the viewport to `<meta name="viewport" content="width=device-width">` without the initial-scale. It can't be done because you get an error if you try to overwrite the metatag in the `<Head>`.

Any ideas?
